cilium-cli/cli/connectivity.go

```diff
@@ -175,6 +175,8 @@ func newCmdConnectivityTest(hooks api.Hooks) *cobra.Command {
 	cmd.Flags().StringVar(&params.ConnDisruptTestRestartsPath, "conn-disrupt-test-restarts-path", "/tmp/cilium-conn-disrupt-restarts", "Conn disrupt test temporary result file (used internally)")
 	cmd.Flags().StringVar(&params.ConnDisruptTestXfrmErrorsPath, "conn-disrupt-test-xfrm-errors-path", "/tmp/cilium-conn-disrupt-xfrm-errors", "Conn disrupt test temporary result file (used internally)")
 	cmd.Flags().DurationVar(&params.ConnDisruptDispatchInterval, "conn-disrupt-dispatch-interval", 0, "TCP packet dispatch interval")
+	cmd.Flags().BoolVar(&params.SkipConnDisruptTestNSTraffic, "skip-conn-disrupt-test-ns-traffic", false, "Skip conn disrupt test for NS traffic")
+	cmd.Flags().MarkHidden("skip-conn-disrupt-test-ns-traffic")
 
```

Comment:
Good call! I'd expect quite a bit of fallout from this new test scenario, we'll definitely need to opt-out easily. And should probably set this opt-out by default on `v1.17` and older, to not disrupt CI there too much.

Reply:
Yeah, I'd expect it too. Maybe we should opt-out this test by default. Otherwise it would block the PRs which bump the cilium-cli on the stable branches